src/olympia/abuse/models.py

```diff
@@ -160,10 +193,7 @@ def target(self):
 
     @property
     def type(self):
-        with translation.override(settings.LANGUAGE_CODE):
-            type_ = (translation.ugettext(amo.ADDON_TYPE[self.addon.type])
-                     if self.addon else 'User' if self.user else 'Addon')
-        return type_
+        return 'User' if self.user else 'Addon'
```

Comment:
So we're dropping the ability to differentiate between different types of Add-on in the emails, admin, etc?  Themes abuse reports would be indistinguishable from Extension reports.

Reply:
Ah, I had not considered the impact to themes. It got in my way because the main thing I needed to differentiate between was users and addons. @wagnerand do you know if we care ? If we do, I can add it back, I'll need to figure out how to implement type filtering in the admin differently...

Reply:
I think it would be good to have since theme admins are not necessarily the same group of people as extension admins. If it's major effort though, let's talk about it.

Reply:
27215b8d382d88841b0bbffdf91cd31be4dc4424 restores it. The drawback is that it can be a little confusing because the admin doesn't let you filter that precisely (it's either addons or users when filtering by type) even though it displays the precise type in the corresponding column.